.travis.yml

```diff
@@ -47,7 +48,8 @@ python:
 #
 env:
   matrix:
-    - PYSMT_SOLVER="all"
+    - PYSMT_SOLVER="all" PYSMT_GMPY="TRUE"
+    - PYSMT_SOLVER="all" PYSMT_GMPY="FALSE"
```

Comment:
Yes, that is correct.
The single solver builds are without, because we do not install gmpy. To be explicit, we could just add ```PYSMT_GMPY="FALSE"``` in all other cases.